packages/experience-builder-sdk/src/hooks/useFetchById.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback } from 'react';
+import type { ContentfulClientApi } from 'contentful';
+import type { ExternalSDKMode } from '@contentful/experience-builder-core/types';
+import { useFetchByBase } from './useFetchByBase';
+import { fetchById } from '@contentful/experience-builder-core';
+
+export type useFetchByIdArgs = {
```

Comment:
[nitpick]
Following our CF-internal naming styles, this would usually be written like this:
```suggestion
export type UseFetchByIdProps = {
```
- A type is always written in UpperCamelCase
- The type describing the props for a hook or component uses always the suffix "Props"

It's really just a nitpick but I think it's a nicer DX if we use similar casing and prefixes/ suffixes for such things :)